integration-tests/config.ts

```diff
@@ -56,69 +56,69 @@ interface FaucetConfig {
   faucetKey: {};
 }
 
-const idiazabalnetEphemeral = {
-  rpc: process.env['TEZOS_RPC_IDIAZABALNET'] || 'https://idiazabalnet.ecadinfra.com',
-  knownBaker: 'tz1cjyja1TU6fiyiFav3mFAdnDsCReJ12hPD',
-  knownContract: 'KT1UUS3M31GpFS4k5xQtp97oWGDhdzdT9mNJ',
-  knownBigMapContract: 'KT1VFM8dyKv2YTFdLBZm8seJKgFnn7RMrDtv',
-  knownTzip1216Contract: 'KT1PiiNe7icQHeaQ4Z8m27FVhYWQ2pPGMots',
-  protocol: Protocols.ProtoALpha,
+const ithacanetEphemeral = {
+  rpc: process.env['TEZOS_RPC_ITHACANET'] || 'http://ecad-ithacanet-full.i.tez.ie:8732',
```

Comment:
Should we use the public endpoint `https://ithacanet.ecadinfra.com/` instead of `http://ecad-ithacanet-full.i.tez.ie:8732` so the users can run the integration tests?

Reply:
changed